### ISSUE TITLE
[suggestion] Set up Official Chatroom

### ISSUE THREAD

**Issue by ntutangyun** [lgsvl/simulator#863]
Hi LGSVL team,

May I suggest that we set up a public chatroom for quick discussions on discord or other similar platforms? 

Thanks

**Comment by rongguodong** [comment]
I thought it is a great idea. Not sure if we have resources to maintain it. We will discuss internally about it.

**Comment by EricBoiseLGSVL** [comment]
@ntutangyun @rongguodong I agree this would be great for users to connect with each other.  We will figure out how to get this going.  Thanks for the suggestion.

**Comment by EricBoiseLGSVL** [comment]
@kanakab We are still discussing this internally.  We would like to get a Discord up and running soon.
